### PR TITLE
Allow closing alterts while blackout with certain severities. Closes …

### PR DESCRIPTION
**Pull request by tdi**
…#176

Severities are configured via BLACKOUT_ACCEPT configuration option,
e.g. BLACKOUT_ACCEPT=['ok', 'normal']

### CODE REVIEW

File: alerta/app/database.py [hunk 1]
```diff
@@ -860,6 +860,11 @@ def get_blackouts(self, query=None):
         return blackouts
 
     def is_blackout_period(self, alert):
+        
+        blackout_accept = app.config.get('BLACKOUT_ACCEPT', None)
+        if blackout_accept:
+            if alert.severity in blackout_accept:
+                return False
```
Comment (satterly) [inline alerta/app/database.py]: I would simplify this as...
```
if alert.severity in app.config.get('BLACKOUT_ACCEPT', []):
    return False
```

**Comment by tdi** [comment]
I have changed the code as you suggested.

**Comment by satterly** [comment]
I assume you have tested this and it works as expected?

**Comment by satterly** [comment]
:+1: Thanks

**Pull request merged**